From b0e6e445b2d6d7eb2acd8692c0f9c470f391f78f Mon Sep 17 00:00:00 2001
From: Sridharan S N <quic_sridsn@quicinc.com>
Date: Mon, 21 Oct 2024 17:43:44 +0530
Subject: [PATCH] usb: gadget: Kconfig: Update USB_CONFIGFS_F_QDSS as tristate

Update USB_CONFIGFS_F_QDSS as tristate to generate it as module.

Change-Id: Ic04444784a37f91667237b1cac91e19821aaa15f
Signed-off-by: Sridharan S N <quic_sridsn@quicinc.com>
---
 drivers/usb/gadget/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/Kconfig b/drivers/usb/gadget/Kconfig
index b89ab8a8698c..73831ecb987b 100644
--- a/drivers/usb/gadget/Kconfig
+++ b/drivers/usb/gadget/Kconfig
@@ -509,7 +509,7 @@ config USB_CONFIGFS_F_TCM
 	  UAS utilizes the USB 3.0 feature called streams support.
 
 config USB_CONFIGFS_F_QDSS
-	bool "USB QDSS function"
+	tristate "USB QDSS function"
 	select USB_F_QDSS
 	depends on USB_CONFIGFS
 	help
-- 
2.34.1

